ci/kokoro/ubuntu/build.sh

```diff
@@ -114,8 +114,7 @@ echo "================================================================"
 echo "Running Google Cloud Bigtable Integration Tests"
 (cd $(bazel info bazel-bin)/google/cloud/bigtable/tests && \
    "${PROJECT_ROOT}/google/cloud/bigtable/tests/run_integration_tests_production.sh")
-# TODO(#730) - the example scripts require running out of `tests`, ugh.
-(cd $(bazel info bazel-bin)/google/cloud/bigtable/tests && \
+(cd $(bazel info bazel-bin)/google/cloud/bigtable/examples && \
```

Comment:
This is part of the fixes, but it is not complete. The real problem is the fact that the path for the executables needs to include the examples directory, for example, here:

https://github.com/GoogleCloudPlatform/google-cloud-cpp/blob/de6898bbf2ceec97f1f609d9a40dcc1d4cb6531b/google/cloud/bigtable/examples/run_examples_utils.sh#L91

Reply:
If I am understanding correct, then I tried below command from tests and examples, and in both case running successfully, so please suggest, if I am going in wrong direction.
cd google-cloud-cpp/build-output/google/cloud/bigtable/examples
bash ../../../../../google/cloud/bigtable/examples/run_examples_emulator.sh

cd google-cloud-cpp/build-output/google/cloud/bigtable/tests
bash ../../../../../google/cloud/bigtable/examples/run_examples_emulator.sh

Reply:
That is great news.  Can you remove the `examples/` path in `bigtable/examples/run_examples_utils.sh` as part of this PR then?

Also fix this path:

https://github.com/GoogleCloudPlatform/google-cloud-cpp/blob/3af81a5fb10f68f9c66d02629c9498e603b8cf75/google/cloud/bigtable/ci/run_integration_tests.sh#L56